Formula/terminally-tetris.rb

```diff
@@ -0,0 +1,24 @@
+class TerminallyTetris < Formula
+  desc "Play Tetris in the Terminal"
+  homepage "https://github.com/thecardkid/terminally-tetris"
+  url "https://github.com/thecardkid/terminally-tetris/archive/v1.0.0.tar.gz"
+  sha256 "213f8a7581e69d3284291187eb887a1c0822fe699d53a7768f0fea777bc1427b"
+
+  depends_on "cmake" => :build
+
```

Comment:
The homepage says it needs `ncurses` (and not the system one), shouldn't it be a dependency here?